src/nxdoc/nuxeo-add-ons/nuxeo-enhanced-viewer.md

```diff
@@ -338,78 +330,189 @@ A new window is opened with the two documents side-by-side, highlighting text wh
+In the end, you should have the following in your `nuxeo.conf`:
 
-You can change the ARender previewer URL used by Nuxeo to open ARender sessions by setting `arender.server.previewer.host` in your `nuxeo.conf` (default value if `http://localhost:8080`).
+```properties
+arender.server.previewer.host=https://arender-previewer-url
+nuxeo.jwt.secret=JWT_SECRET
+nuxeo.arender.secret=OAUTH2_SECRET
+```
 
-### ARender Previewer Configuration
+#### With and after 10.3
 
-- For an on-host installation, you can follow the [ARender Documentation](https://arender.io/doc/current4/documentation/setup/presentation/configuration.html).
+The Nuxeo Enhanced Viewer relies on a complete OAuth2 challenge for authentication (See [OAuth 2]({{page page='using-oauth2'}}) documentation page for more information). Thus you need to define an OAuth2 client, with shared secret and redirect URI, under the `nuxeo.arender.oauth2.client.` prefix.
 
-- For an embedded installation, you can modify the `arender-hmi.properties` files under `NUXEO_HOME/nxserver/config/ARenderConfiguration` folder.
+In the end, you should have the following in your `nuxeo.conf`:
 
-- For a Docker installation, you can extend our image and copy your properties file to `/docker-entrypoint-init.d/arender.properties`:
+```properties
+arender.server.previewer.host=https://arender-previewer-url
+nuxeo.arender.oauth2.client.create=true
+nuxeo.arender.oauth2.client.id=arender
+nuxeo.arender.oauth2.client.secret=OAUTH2_SECRET
+nuxeo.arender.oauth2.client.redirectURI=/login/oauth2/code/nuxeo
+```
 
-  ```
-  FROM dockerin-arender.nuxeo.com:443/arender-previewer:MP_VERSION
+### ARender Configuration
 
-  COPY arender.properties /docker-entrypoint-init.d/arender.properties
-  ```
+Nuxeo Enhanced Viewer involves to install the ARender services. You can install the ARender services using Kubernetes and Helm 3 by following [ARender Documentation](https://arender.io/v4/documentation/install/kubernetes/).
 
-You can also modify the `arender-hmi.properties` which is deployed inside the `/ARenderConfiguration` in the previewer Docker container.
+{{#> callout type='warning' heading='Private addon'}}
+You should contact your Nuxeo Administrator or your Nuxeo sales representative to get access to these artefacts.
+{{/callout}}
+
+#### Common
+
+After getting the ARender Helm Chart, you need to customize some part in order to deploy the Nuxeo connector and add configuration. You can find below an example of _values.yaml_:
+
+```yaml
+web-ui:
+  image:
+    repository: docker-arender.packages.nuxeo.com/nuxeo/arender-previewer
+    tag: [NEV_VERSION]
+  environment:
+    # required values
+    ARENDERSRV_NUXEO_SERVER_URL: https://nuxeo-host/nuxeo
+    ARENDERSRV_NUXEO_SERVER_ARENDER_SECRET: OAUTH2_SECRET # Same than the one in nuxeo.conf
+    # optional values with their defaults
+    ARENDERSRV_NUXEO_CLIENT_TIMEOUT: 30
+    ARENDERSRV_NUXEO_SERVER_CONTEXT_PATH: ''
```

Comment:
Is it really needed? the context path is already in `ARENDERSRV_NUXEO_SERVER_URL` (?)

Reply:
It's an optional value to fill. Depending on your setup you may need this. It's our case in the CI because the URL is automatically injected by the exposecontroller without the context path.
Do you think it's better to not mention it?

Reply:
`ARENDERSRV_NUXEO_SERVER_CONTEXT_PATH` has now `/nuxeo` as default, documentation has been updated to reflect this change.